.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,37 @@
+name: auth-landing CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
This would cause the deploy steps to run twice, since the matrix generates two jobs, one for each node version. We can just remove it for now, since in our case just testing on one node version for building is enough.

As a suggestion, I think a cleaner way would be using two jobs:
- `test`: run the tests (lint & format check) and build the bundle.
- `deploy-master`: depends on `test` step; build the bundle and deploy to `gh-pages`.

Reply:
good advice 👍